util/gh-pages/index_template.html

```diff
@@ -0,0 +1,262 @@
+                    <header class="panel-heading" onclick="expandLint('{(lint.id)}')">
+                        <h2 class="panel-title">
+                            <div class="panel-title-name" id="lint-{(lint.id)}">
+                                <span>{(lint.id)}</span>
+                                <a href="#{(lint.id)}" class="anchor label label-default" onclick="openLint(event)">&para;</a>
+                                <a href="" class="anchor label label-default" onclick="copyToClipboard(event)">
+                                    &#128203;
+                                </a>
+                            </div>
+
+                            <div class="panel-title-addons">
+                                <span class="label label-lint-group label-default label-group-{(lint.group)}">{(lint.group)}</span>
+
+                                <span class="label label-lint-level label-lint-level-{(lint.level)}">{(lint.level)}</span>
+
+                                <span class="label label-doc-folding">&plus;</span>
+                            </div>
+                        </h2>
+                    </header>
+
+                    <div class="list-group lint-docs">
+                        <div class="list-group-item lint-doc-md">{(Self::markdown(lint.docs))}</div>
+                        <div class="lint-additional-info-container">
+                            {# Applicability #}
+                            <div class="lint-additional-info-item">
+                                <span> Applicability: </span>
+                                <span class="label label-default label-applicability">{( lint.applicability_str() )}</span>
+                                <a href="https://doc.rust-lang.org/nightly/nightly-rustc/rustc_lint_defs/enum.Applicability.html#variants">(?)</a>
+                            </div>
+                            {# Clippy version #}
+                            <div class="lint-additional-info-item">
+                                <span>{% if lint.group == "deprecated" %}Deprecated{% else %} Added{% endif %} in: </span>
+                                <span class="label label-default label-version">{(lint.version)}</span>
+                            </div>
+                            {# Open related issues #}
+                            <div class="lint-additional-info-item">
+                                <a href="https://github.com/rust-lang/rust-clippy/issues?q=is%3Aissue+{(lint.id)}">Related Issues</a>
+                            </div>
+
+                            {# Jump to source #}
+                            {% if let Some(id_location) = lint.id_location %}
+                                <div class="lint-additional-info-item">
+                                    <a href="https://github.com/rust-lang/rust-clippy/blob/master/clippy_lints/{(id_location)}">View Source</a>
+                            {% endif %}
+                            </div>
```

Comment:
![image](https://github.com/user-attachments/assets/ee0ecf1b-eaf3-4e98-8e49-85e869992344)


```suggestion
                                </div>
                            {% endif %}
```